carla_integration/VehicleConfigParams.yaml

```diff
@@ -67,35 +77,41 @@ vehicle_steering_gear_ratio: 16.863
 # Unit: deg
 vehicle_steer_lim_deg: 29.2
 
-# steering dynamics time constant 
-# Value type: Measured 
+# steering dynamics time constant
+# Value type: Measured
 # Unit: s
 vehicle_model_steer_tau : 0.3
 
 # Parameter to switch between passenger car and truck
 # Value type: Measured
 truck: false
-car: true 
+car: true
 
 # Required drivers for the vehicle to be functional
 required_drivers:
-  - /hardware_interface/carla_ackermann_control_wrapper
+  - /hardware_interface/carla_driver
```

Comment:
Isn't the carma-cohda dsrc driver required?

Reply:
Good catch, just added it and tested it.